src/config/graphical-session/icons.md

```diff
@@ -0,0 +1,18 @@
+# Icons
+
+## GTK
+
+In the absence of any other configuration (i.e. via `/etc/gtk-3.0/settings.ini`
+or `~/.config/gtk-3.0/settings.ini`), GTK-based applications will try to use the
+Adwaita icon theme for application icons. However, as users might prefer to use
+a different theme, the `adwaita-icon-theme` package is not installed by default,
+resulting in GTK applications not displaying the appropriate icons. If you have
+no preference for a different icon theme, install the `adwaita-icon-theme`
+package to ensure application icons are displayed.
```

Comment:
This feels like something that should be solved in packaging. I would like to raise it again with @void-linux/pkg-committers

Reply:
Yes, my preference would be for the GTK packages (certainly `gtk+3`, maybe `gtk+`?) to pull in `adwaita-icon-theme`.

Reply:
We'll see if anybody objects to `gtk+3` depending on `adwaita-icon-theme` in [void-packages #23530](https://github.com/void-linux/void-packages/pull/23530). I think I was bitten by this when setting up my first Void system.

If everybody agrees and the dependency is added, the docs should be updated to note that the theme can be removed after an `ignorepkg` provided users are willing to manage their own themes.

Reply:
That seems like a good approach, thanks.